akka-cluster-sharding/src/main/mima-filters/2.5.x.backwards.excludes

```diff
@@ -3,3 +3,6 @@ ProblemFilters.exclude[Problem]("akka.cluster.sharding.Shard.*")
 
 # #25191
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.sharding.ShardRegion.retryTask")
+
+# #27273 Configurable shard query timeout
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.sharding.ClusterShardingSettings.this")
```

Comment:
Since `ClusterShardingSettings` is public API, I don't think we can exclude it like this.

Instead, we can add a constructor just like the one at https://github.com/akka/akka/pull/27274/files#diff-797acfe9176d356b2293f2e1bf69ecc3R230 that keeps the 'old' constructor signature (without the timeout parameter) available next to the modified one.

Reply:
Thanks, pushed.